components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41Utils.java

```diff
@@ -1188,12 +1180,12 @@ public static void removeServiceGroup(String name) throws RestAPIException, Auto
 
             // Remove the dependent cartridges and cartridge groups from Stratos Manager cache
             // - done after service group has been removed
-            if (serviceGroup.getCartridges() != null) {
-                List<String> cartridgeList = new ArrayList<String>();
-                findCartridgesInServiceGroup(serviceGroup, cartridgeList);
-                String[] cartridgeNames = cartridgeList.toArray(new String[cartridgeList.size()]);
-                smServiceClient.removeUsedCartridgesInCartridgeGroups(name, cartridgeNames);
-            }
+
+            List<String> cartridgeList = new ArrayList<String>();
+            findCartridgesInServiceGroup(serviceGroup, cartridgeList);
```

Comment:
Is this method using the second parameter (cartridgeList) to return the result? If so why not directly return it from the method?